controlpanel/frontend/views/app.py

```diff
@@ -248,9 +247,9 @@ class RemoveCustomer(UpdateApp):
 
     def perform_update(self, **kwargs):
         app = self.get_object()
-        user_id = self.request.POST.get('customer')
-        app.delete_customers([user_id])
-        messages.success(self.request, "Successfully removed customer")
+        user_ids = self.request.POST.getlist('customer')
```

Comment:
not sure what's the convention but maybe name this `customers` as it's a list really? 🤷‍♂

Reply:
it's not a list, it's like the `customer` form param being submitted multiple times. `.get()` will just take the first one, `.getlist()` will give you every time it has been submitted in the current request. I think the name is right here.

Reply:
happy for you to leave it as it 👍 (still looks like `customer` contains the customer IDs to bulk delete, submitted in the POST request being handled 🤷‍♂)

Reply:
@xoen `customer` is the name of the form field. There is more than one field, but each field's name is `customer`. It wouldn't make sense to call each field `customers`, as you only populate the field with a single customer user ID.